testsuite/pytests/mpi/2/test_issue_578_sp.py

```diff
@@ -19,18 +19,20 @@
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
-"""
-This test is called from test_mpitests.py
-"""
 
 import nest
 import sys
+import unittest
 
 HAVE_GSL = nest.ll_api.sli_func("statusdict/have_gsl ::")
 
 
 class TestIssue578():
+    """
+    This test must be run with 2 mpi processes.
+    """
```

Comment:
I think this comment should rather explain what the test is about and the condition should be turned into a `skipIf` to check if we actually have two processes.

Reply:
If a test in `mpi/2/` is run with anything other than two processes, should it be skipped or raise an error?

Reply:
I think that should be up to the test. I can imagine situations, where you would symlink a test into multiple directories to get it run with multiple numbers of processes.

But I agree that violations should probably be treated as error or faillure instead of skip.

Reply:
I have renamed the test to the correct issue number (#576), implemented skipping for insufficient processes, and simplified the test code a little.